src/components/jobs/JobsList.jsx

```diff
@@ -28,40 +29,26 @@ const displayJobs = (studiesFilter, typesFilter, jobs) => {
 
 const createJobs = jobbies => {
   return jobbies.map(job => (
-    <JobItem
-      className="item"
-      key={job.node.id}
-      job={job.node}
-      partner={job.node.partner}
-    />
+    <Grid.Column>
+      <JobItem key={job.node.id} job={job.node} partner={job.node.partner} />
+    </Grid.Column>
   ));
 };
 
 const jobslist = props => (
   <JobsList>
     <div className="container">
-      {displayJobs(props.studiesFilter, props.typesFilter, props.jobs)}
+      <Grid doubling stackable stretched columns={3}>
+        {displayJobs(props.studiesFilter, props.typesFilter, props.jobs)}
+      </Grid>
     </div>
   </JobsList>
 );
 
 const JobsList = styled.div`
```

Comment:
Please rename this to `JobsWrapper` akin to the other wrappers in the application so the main component from this file `jobslist` can be renamed to `JobsList`.

Reply:
This has been called JobsList since the beginning of this school year :P. Will do though